app/models/organisation.rb

```diff
@@ -1,14 +1,20 @@
 class Organisation < ApplicationRecord
   has_many :users, inverse_of: :organisation
-  has_many :ips
+  has_many :locations
+  has_many :ips, through: :locations
 
   validates :name, presence: true, uniqueness: true
 
   before_create :set_radius_secret_key
+  after_create :create_location
```

Comment:
hack hack hack

Reply:
Might be good to call this `create_default_location` or something like that, try and make it a bit clearer that it's the default bucket things fall into.